index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Joseph Kausi'Portfoliot</title>
+</head>
+<body>
+    <header id="main-head" class="family">
+        <h1>WELCOME</h1>
+        <a href=""><img src="image/Icon-Menu.png" alt="Icon-Menu" id="icn-menu"></a>
+    </header>
+    <section class="about-me family">
+        <h2 class="me">Hey there. I’m Mariane
+            <span class="whatIdo">I’m a software developer</span></h2>
+
+        <p class="paragrph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+
+        <section id="social-media">
+            <a href="#"><img src="image/twitter.png" alt="twitter"></a> 
+            <a href=""><img src="image/Linkedin icon.png" alt="linkedIn"></a> 
+            <a href=""><img src="image/medium.png" alt="medium"></a>
+            <a href=""><img src="image/github.png" alt="github"></a>
+            <a href=""><img src="image/angellist.png" alt="angellist"></a>
+        </section>
+        <a href=""></a><img src="image/Action.png" alt="action-button" id="more">
```

Comment:
The distance between the social media icons and the down button is more than what you currently have. See image below: 

Your design | Figma design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/205169502-68fe9da9-1d26-4956-9e7f-0138d0ae6fb1.png) | ![image](https://user-images.githubusercontent.com/84156714/205169590-b78c2ac6-a279-4790-885a-9c04f73e57d3.png)